src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -40,13 +40,19 @@
 long-summary: >
     Update the properties of a storage account's blob service, including
     properties for Storage Analytics and CORS (Cross-Origin Resource
-    Sharing) rules. But currently we only support enabling or disabling change feed for a storage account's blob service.
+    Sharing) rules.
 parameters:
   - name: --enable-change-feed
     short-summary: 'Indicate whether change feed event logging is enabled. If it is true, you enable the storage account to begin capturing changes. The default value is true. You can see more details in https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-change-feed?tabs=azure-portal#register-by-using-azure-cli'
+  - name: --enable-delete-retention
+    short-summary: 'Indicate whether delete retention policy is enabled for the blob service.'
+  - name: --delete-retention-days
+    short-summary: 'Indicate the number of days that the deleted blob should be retained. The minimum specified value can be 1 and the maximum value can be 365. It must be provided when `--enable-delete-retention` is true.'
```

Comment:
can we use a range like [1, 365] here? it is a little long

Reply:
Yeah, I changed it.